lib/builderator/control/version.rb

```diff
@@ -32,6 +32,12 @@ class Version
       class << self
         def current
           @current ||= SCM.tags.last
+
+          if @current.nil? && Util.relative_path('VERSION').exist?
```

Comment:
Should we leave this? `@current ||= ...` will already perform this check in a way. If `@current` is non-nil it will not re-assign it.